diff --git a/node_modules/react-linkify/dist/decorators/defaultMatchDecorator.js b/node_modules/react-linkify/dist/decorators/defaultMatchDecorator.js
index de684da..924743e 100644
--- a/node_modules/react-linkify/dist/decorators/defaultMatchDecorator.js
+++ b/node_modules/react-linkify/dist/decorators/defaultMatchDecorator.js
@@ -6,7 +6,7 @@ Object.defineProperty(exports, "__esModule", {
 
 var _linkifyIt = require('linkify-it');
 
-var _linkifyIt2 = _interopRequireDefault(_linkifyIt);
+var _linkifyIt2 = { default: _linkifyIt.LinkifyIt || _linkifyIt.default || _linkifyIt };
 
 var _tlds = require('tlds');
 
